components/http_foundation/session_configuration.rst

```diff
@@ -45,8 +45,8 @@ Example usage::
     use Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage;
     use Symfony\Component\HttpFoundation\Session\Storage\Handler\NativeFileSessionHandler;
 
-    $storage = new NativeSessionStorage(array(), new NativeFileSessionHandler());
-    $session = new Session($storage);
+    $nativeSessionStorage = new NativeSessionStorage(array(), new NativeFileSessionHandler());
```

Comment:
I would rename it to `$sessionStorage` because "native" or not is just an implementation detail that can change at any moment.

Reply:
Agree.